### DISCUSSION TITLE
想问一下怎么运行example中的测试代码

### DISCUSSION THREAD

**Discussion by Raymond-Reddington** [chenshuo/muduo#575] [General]
GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, send it to mailing list:

muduo-library@googlegroups.com

Or visit:
https://groups.google.com/forum/#!forum/muduo-library

When file an issue muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

If you can't compile muduo, make sure you install `cmake` and `boost` from the
official package repository, e.g. `apt` or `yum`, before opening a bug.
Don't open a bug if you installed boost from a third-party source or
downloaded it by yourself, and couldn't compile muduo, thank you.

Also specify the exact environment where the issue occurs:

**Comment by fortunely** [comment]
编译后，根目录或者cmake-build-debug下面，会有个bin目录，里面有你生成的所有可执行文件。
终端下输入运行程序命令就可以了。
